<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanWidget</class>
 <widget class="QWidget" name="ScanWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="title">
     <property name="text">
      <string notr="true">Title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="scanPaths">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Drag and drop files and/or directories here to add them to the scan.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideMiddle</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="removeScanPath">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove the selected path from the list to scan.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="list-remove">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="chooseScanFiles">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add a file to the list of paths to scan.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="text-plain">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="chooseScanDirectory">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add a directory to the list of paths to scan.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="folder">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="saveScanProfile">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create a new scan profile with the listed paths, or update the selected profile based on your current changes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="document-save">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QProgressBar" name="scanProgress"/>
         </item>
         <item>
          <widget class="QPushButton" name="abortButton">
           <property name="toolTip">
            <string>Abort the scan.</string>
           </property>
           <property name="text">
            <string>Abort</string>
           </property>
           <property name="icon">
            <iconset theme="dialog-cancel">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="scanStatus">
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="issuesListLabel">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Issues</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="issuesList">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Infections found during the last scan.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideMiddle</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>200</number>
         </attribute>
         <attribute name="headerMinimumSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="scanButton">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start scanning the listed paths.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Scan</string>
           </property>
           <property name="icon">
            <iconset theme="system-search">
             <normaloff/>
            </iconset>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>scanButton</sender>
   <signal>clicked()</signal>
   <receiver>ScanWidget</receiver>
   <slot>doScan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>abortButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>ScanWidget</receiver>
   <slot>abortScan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeScanPath</sender>
   <signal>clicked()</signal>
   <receiver>ScanWidget</receiver>
   <slot>removeSelectedScanPaths()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chooseScanDirectory</sender>
   <signal>clicked()</signal>
   <receiver>ScanWidget</receiver>
   <slot>chooseScanDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chooseScanFiles</sender>
   <signal>clicked()</signal>
   <receiver>ScanWidget</receiver>
   <slot>chooseScanFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scanPaths</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>ScanWidget</receiver>
   <slot>slotScanPathsSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>scanButtonClicked()</signal>
  <signal>saveProfileButtonClicked()</signal>
  <slot>chooseScanDirectory()</slot>
  <slot>chooseScanFiles()</slot>
  <slot>doScan()</slot>
  <slot>abortScan()</slot>
  <slot>removeSelectedScanPaths()</slot>
  <slot>scanProfileChanged()</slot>
  <slot>removeCurrentScanProfile()</slot>
  <slot>slotScanPathsSelectionChanged()</slot>
 </slots>
</ui>
